src/FeeModule.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity ^0.8.24;
+
+import {OwnableUpgradeable, IERC4626, ERC20} from "src/Common.sol";
+import {IVault} from "src/interface/IVault.sol";
+import {YnETHx} from "src/YnETHx.sol";
+import {IFeeModule} from "src/interface/IFeeModule.sol";
+
+contract FeeModule is OwnableUpgradeable, IFeeModule {
+    // performance denominated in ether(i.e. 1e18 = 100%)
+    uint256 public performanceFee;
+    address public performanceFeeRecipient;
+    IVault public immutable VAULT;
+    uint256 public immutable VAULT_DECIMALS;
+    uint256 public maximumAccountedExchangeRate;
+
```

Comment:
assuming we don't do HWM or using the all-time-high max,


consider what arugments you need to pass into the `afterProcessAccounting` call to make it really easy to calculate things here and simplify all computations

I think it won't need any extra storage